.pre-commit-config.yaml

```diff
@@ -51,23 +45,6 @@ repos:
     hooks:
       - id: codespell
         additional_dependencies: ["tomli"]
-  - repo: https://github.com/PyCQA/flake8
-    rev: 6.0.0
-    hooks:
-      - id: flake8
-        additional_dependencies:
-          - flake8-bugbear
-          - flake8-comprehensions
-          - flake8-debugger
-          - flake8-string-format
-          - flake8-typing-imports
-  - repo: https://github.com/PyCQA/bandit
-    rev: '1.7.5'
-    hooks:
-    - id: bandit
-      args: ["-c", "pyproject.toml", "--quiet", "--format=custom"]
-      files: ^dvc/
-      additional_dependencies: [".[toml]"]
```

Comment:
There are some shell/subprocess specific checks that are absent in ruff.

https://github.com/charliermarsh/ruff/issues/1646

Reply:
Do you think those are critical? Is it worth keeping it or would they get implemented soon?

Reply:
Seems that we are explicitly ignoring in the code the occurrences of some of those checks

Reply:
Let's wait for a new ruff release. Looks like some checks were added.